tests/debuginfo/macro-stepping.rs

```diff
@@ -26,6 +31,8 @@ extern crate macro_stepping; // exports new_scope!()
 // gdb-command:frame
 // gdb-check:[...]#loc3[...]
 // gdb-command:next
+// FIXME: for some reason we need two `next` to skip over the `vec!`.
+// gdb-command:next
```

Comment:
I have now made the test pass by adding a second `next` invocation here... but clearly that's not the actually intended behavior. I just have no idea how to make the debugger jump over the entire thing in a single step again.

Reply:
Seems like maybe only lldb needs the second `next`, and on gdb it works without?